homeassistant/helpers/deprecation.py

```diff
@@ -138,6 +138,34 @@ def deprecated_func(*args: _P.args, **kwargs: _P.kwargs) -> _R:
     return deprecated_decorator
 
 
+def deprecated_hass_binding[**_P, _T](
```

Comment:
I think the name is misleading since this is not about deprecating `bind_hass`. Maybe rename to `deprecated_hass_argument` or something like that?

Reply:
> I think the name is misleading since this is not about deprecating `bind_hass`. Maybe rename to `deprecated_hass_argument` or something like that?

Renamed